Build_iOS/README.md

```diff
@@ -1 +1 @@
-Please consult the documentation [here](https://github.com/Microsoft/cpprestsdk/wiki/How-to-build-for-iOS) for iOS compilation.
+Please consult the [documentation for iOS compilation](https://github.com/Microsoft/cpprestsdk/wiki/How-to-build-for-iOS).
```

Comment:
This is based on an accessibility complaint (for people using screen readers):

#### Do not use `(click) here` links

For more information, see:
* https://www.w3.org/QA/Tips/noClickHere
* https://webaim.org/techniques/hypertext/link_text
* https://granicus.com/blog/why-click-here-links-are-bad/
* https://heyoka.medium.com/dont-use-click-here-f32f445d1021
```
(?i)(?:>|\[)(?:(?:click |)here|link|(?:read |)more)(?:</|\]\()
```